### ISSUE TITLE
Need a method to reset password - and fast

### ISSUE THREAD

**Issue by PawWriter** [Benjamin-Loison/freeotp-android#6]
TL;DR - Is there any way to reset the master password for FreeOTP?

Ok, so I straight up joined GitHub solely to post this here.

After the upgrade to FreeOTP 2.0, I set a password, stored it in a password keeper, and figured I was good to go.

Fast forward, I now have to replace my phone (where FreeOTP is installed) due to a defect in the screen.  Export goes fine, but of all the crap things - the password keeper database is corrupted.  Thus, my master password for FreeOTP is gone.

I now have exactly five days before I have to ship my old phone back for the warranty.  I need to be able to restore something like 20 OTP keys.  I need a way to reset my master password for the app so I can export my keys from the old phone.

Note: I am in no way a programmer, though I can follow tech instructions very well.  I'm just your run of the mill consumer - and I'm willing to bet I'm not the only one who could use a password reset mechanism.

**Comment by johnnynoone** [comment]
No idea about your problem, but this is not the official repository for FreeOTP. This is just a fork. The official repo is here: https://github.com/freeotp/freeotp-android
